pangeo_forge/pipelines/job.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: prefect-dask-job
+  labels:
+    app: prefect-dask-job
+spec:
+  template:
+    metadata:
+      labels:
+        app: prefect-dask-job
+    spec:
+      serviceAccountName: pangeo-forge
+      serviceAccount: pangeo-forge
+      containers:
+        - name: flow
+          image: pangeoforge/noaa-oisst-avhrr
```

Comment:
How do we get around the hard-coding of this image name? Can we set these things on the fly when we run the flow?

Reply:
OK, we're no longer hardcoding the image here. We set the image at https://github.com/pangeo-forge/pangeo-forge/pull/14/files#diff-467822c6f6378f68bea635c429827a2caf36c7f16cb25944cc7b5146262cf35aR105. I'm hopeful that we can layer on additional packages if you just need to `pip install` one more thing.